website/docs/reference/targets.md

```diff
@@ -21,13 +21,18 @@ The library target accepts following configuration parameters, with default valu
 
 ```toml
 [lib]
-sierra = true  # Enable Sierra codegen.
+sierra = true  # Enable JSON Sierra codegen.
 casm = false   # Enable CASM codegen.
+sierra-text = false # Enable textual Sierra codegen.
 ```
 
-By default, the library target builds unprocessed Sierra code in textual form for the package.
+By default, the library target builds unprocessed Sierra code in JSON form for the package.
 When the `casm` option is enabled, Scarb can also automatically compile this Sierra code down to CASM.
 
+:::warning
+Textual Sierra is being discouraged, try using JSON Sierra instead.
+:::
+
```

Comment:
I don't think we need to shout at the users with a warning box (the less boxes, the generally better, because people will actually pay attention to these if there are few of them). Adding just a single sentence saying that the textual Sierra is fine for debugging would be cool IMHO